CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ This project uses the changelog in accordance with [keepchangelog](http://keepac
  - Fixed `SimulateTagLowFrequencyEx` ignoring the `ledcontrol` argument (@zabszk)
  - Added ledcontrol argument to LF operations (@zabszk)
  - Added new standalone mode `dankarmulti` -  select and run multiple standalone modes (@dankar)
+ - Added new standalone mode `hf_15693SNIFF` - Same as `hf_14ASNIFF` standalone mode for RDV4 - flashmem
```

Comment:
don´t  forget this one and move it to the top of the file

Reply:
Done